### DISCUSSION TITLE
manager/basic-concepts-and-troubleshooting/Basic-Concepts/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4610] [Content Help کمک به اصلاح محتوا]
# manager/basic-concepts-and-troubleshooting/Basic-Concepts/

The proxy manager platform.

https://hiddify.com/manager/basic-concepts-and-troubleshooting/Basic-Concepts/

**Comment by haitovs** [comment]
Hi. how can i set allowInsecure = true?
